### PR TITLE
New Animation added

### PR DESCRIPTION
**Pull request by Haidery**
This solves #43 and #44.

### CODE REVIEW

File: RZTransitions/Transitions/RZZoomFromPointAnimationController.h [hunk 1]
```diff
@@ -0,0 +1,40 @@
+//
+//  RZAnimationControllerProtocol.h
```
Comment (mr-fixit) [inline RZTransitions/Transitions/RZZoomFromPointAnimationController.h]: wrong file name

File: RZTransitions/Transitions/RZZoomFromPointAnimationController.m [hunk 2]
```diff
@@ -0,0 +1,174 @@
+//
+//  RZAnimationControllerProtocol.h
```
Comment (mr-fixit) [inline RZTransitions/Transitions/RZZoomFromPointAnimationController.m]: #req fix copy-paste errors in header comment, a) file name, b) copyright year

**Comment by mr-fixit** [comment]
@Haidery  thanks for your PR! 

I'm going to provide some feedback on your PR using the following key:
#req - Required, i.e. this PR will not be merged until this is fixed. 
#nth - Nice to have, i.e. this would be nice to have now, but will not block merge. Fix eventually. 
#pp - Personal preference, i.e. "I would have done it this way, but this is acceptable"
#? - Question, i.e. "Why did you do it this way?"

If we decide to merge this, I'll ask you look at the [RZ Cocoa style guide](https://github.com/Raizlabs/Raizlabs-Cocoa-Style/blob/master/README.md) and address a few consistent style issues I've seen.

**Comment by mr-fixit** [comment]
@Haidery This transition doesn't work on iOS 8 or 9, and I don't have the setup to test it on iOS7. Have you tried it with Xcode 7?

**Comment by mr-fixit** [comment]
@Haidery This transition doesn't work on iOS 8 or 9, and I don't have the setup to test it on iOS7. Have you tried it with Xcode 7?  Feel free to modify the demo-app to show a working version.  When you do, please submit another PR, against the 'develop' branch.

**Comment by hammad** [comment]
Ok this worked perfectly for me with just minor timings adjustments.

**Pull request closed without merging**